components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -3938,17 +3944,22 @@ def createOriginalFileFromLocalFile(self, localPath,
                                         OriginalFile. If None, use localPath
         :param mimetype:                The mimetype of the file. String.
                                         E.g. 'text/plain'
-        :param ns:                      The namespace of the file.
+        :param ns:                      Deprecated in 5.4.0. This is ignored
         :return:                        New :class:`OriginalFileWrapper`
         """
+        if ns is not None:
+            warnings.warn(
+                "Deprecated in 5.4.0. The ns parameter was added in error"
+                " and has always been ignored",
+                DeprecationWarning)
         if origFilePathAndName is None:
             origFilePathAndName = localPath
         path, name = os.path.split(origFilePathAndName)
         fileSize = os.path.getsize(localPath)
         fileHandle = open(localPath, 'rb')
         try:
```

Comment:
since you are modifying that code you may also want to use ``with open`` instead of the ``try/finally``

Reply:
@will-moore will you adjust the code in this PR?

Reply:
Fixed.